README.md

```diff
@@ -11,6 +11,7 @@ geostationary satellite imagery.
   - [Instructions](#Instructions)
   - [Requirements](#Requirements)
 - [goes/](#goes/)
+- [NAS](#NAS)
```

Comment:
Maybe adding the subsections too? like `Storage`

Reply:
Added!